docs/src/css/custom.css

```diff
@@ -0,0 +1,25 @@
+/* stylelint-disable docusaurus/copyright-header */
```

Comment:
docusaurus boilerplate - this comes part of the "classic" theme

Reply:
i didn't really do any design work, i just swapped in Lyra "purple" into the classic theme.

We can do some work centralizing styles between /docs and /extension in the future, but didn't want to get into that now